src/Modules/Backend/Domain/User/Command/DisableTwoFactorAuthenticationHandler.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\Domain\User\Command;
+
+use ForkCMS\Core\Domain\MessageHandler\CommandHandlerInterface;
+use ForkCMS\Modules\Backend\Domain\User\UserRepository;
+
+class DisableTwoFactorAuthenticationHandler implements CommandHandlerInterface
+{
+    public function __construct(private readonly UserRepository $userRepository)
+    {
+    }
+
+    public function __invoke(DisableTwoFactorAuthentication $disableTwoFactorAuthentication): void
```

Comment:
don't forget to refactor this afterwards

Reply:
fixed d533e75a6455d4bcc45374065603ef440348529e